canonical-websites/www.ubuntu.com
# Removed local /containers/lxd page

## Done

- removed from the footer nav - navigation.yaml
- updated all the redirects to linuxcontainers.org - redirects.yaml
- updated links in
  - /containers/docker-ubuntu
  - /containers
  - /server
- deleted the page - templates/containers/lxd.html
- archived the [copy doc]()

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/containers](http://0.0.0.0:8001/containers) - see that lxd page goes to linuxcontainers.org and the footer and menu no longer have lxd in it 
- go to the [lxd](http://0.0.0.0:8001/containers/lxd) page and see that you go to linuxcontainers.org

## Issue / Card

Partly fixes #3282
